src/wasi_libc.zig

```diff
@@ -34,6 +38,7 @@ pub fn getEmulatedLibCRTFile(lib_name: []const u8) ?CRTFile {
 
 pub fn emulatedLibCRFileLibName(crt_file: CRTFile) []const u8 {
     return switch (crt_file) {
+        .libdl_a => "libdl.a",
```

Comment:
I suppose this doesn't really matter much in the grand scheme of things, but is there a reason why the name doesn't follow the `libwasi_emulated_<thing>.a` pattern?

Reply:
The decision was taken in `wasi-libc`, and we should probably try to remain compatible.

I guess the intent was to make it easier to port existing software because `dl` is a common library, while the rest is wasi-specific.